SplunkforPaloAltoNetworks/default/data/ui/views/iot_security.xml

```diff
@@ -1,14 +1,15 @@
 <form>
   <label>IoT Security</label>
   <fieldset autoRun="true" submitButton="true">
-    <input type="time" searchWhenChanged="true">
+    <input type="time" searchWhenChanged="true" token="time">
       <default>
         <earliest>-24h@h</earliest>
         <latest>now</latest>
       </default>
     </input>
     <input type="multiselect" token="risk_level">
       <label>Risk Level</label>
+      <choice value="*">All</choice>
```

Comment:
Since this is a multiselect, not a single select, I was under the impression that nothing selected would remove the prefix/suffix which would be like an `All` search. Am I wrong about that?